### PR TITLE
fix: change user agent to 'kotlin'

### PR DESCRIPTION
**Pull request by nand4011**
Delete invalid release please changelog.

### CODE REVIEW

**Review by bruuuuuuuce — APPROVED** [review]
:shipit:

File: src/commonMain/kotlin/software/momento/kotlin/sdk/internal/UserHeaderInterceptor.kt [hunk 1]
```diff
@@ -3,7 +3,7 @@ package software.momento.kotlin.sdk.internal
 import kotlin.jvm.Volatile
 
 internal class UserHeaderInterceptor(private val tokenValue: String) : io.grpc.ClientInterceptor {
-    private val sdkVersion = String.format("java:%s", this.javaClass.getPackage().implementationVersion)
+    private val sdkVersion = String.format("kotlin:%s", this.javaClass.getPackage()?.implementationVersion ?:  "unknown")
```
Comment (cprice404) [inline src/commonMain/kotlin/software/momento/kotlin/sdk/internal/UserHeaderInterceptor.kt:6]: can we distinguish between android and jvm versions?

Comment (nand4011) [reply src/commonMain/kotlin/software/momento/kotlin/sdk/internal/UserHeaderInterceptor.kt:6]: Yeah. I can pass that in from the grpc managers that instantiate it.

**Review by bruuuuuuuce — APPROVED** [review]
:shipit:

**Review by cprice404 — APPROVED** [review]
android

**Pull request merged**